persist/sqlite/consensus_refactored_test.go

```diff
@@ -1556,10 +1460,246 @@ func TestFileContractMissed(t *testing.T) {
 		testutil.Equal(t, "len(fces)", 0, len(fces))
 	}
 
+	n.assertContractRevisions(t, fce.ID)
+}
+
+func signRevisions(cs consensus.State, pk types.PrivateKey, txn *types.Transaction) {
+	appendSig := func(key types.PrivateKey, pubkeyIndex uint64, parentID types.Hash256) {
```

Comment:
inline the helper, only used once

Reply:
It's used in TestFileContractRevision and TestFileContractMultipleRevisions